profiles/base-empty.yaml

```diff
@@ -45,4 +45,5 @@ empty_values:
   hunting_buddies: []
   empty_hunting_room_messages: []
   classes_to_teach: []
-  whirlwind_trainables: []
\ No newline at end of file
+  whirlwind_trainables: []
+  hide_type: hide
```

Comment:
Hmm...

@rcuhljr suggested i add hide_type: hide  to the base-empty.   But he wasn't sure if maybe it was a better idea to just let it error out.

Or are you talking about the newline errors that I keep getting? Quite annoying!